bindings/pydeck-carto/pydeck_carto/sources.py

```diff
@@ -144,17 +163,19 @@ class VectorTilesetSourceOptions(TilesetSourceOptions):
 def vector_table_source(**kwargs: Unpack[VectorTableSourceOptions]):
     return pdk.types.Function(
         "vectorTableSource", **{**column_options(**kwargs), **table_options(**kwargs)}
-    )
+    ).serialize()  # TODO Required?
```

Comment:
Preferably this serialization should be handled somewhere on the pydeck side, but from the user's perspective this is an invisible workaround. I'm nervous to make a serialization change to pydeck, so I think I prefer this workaround for now.